api/orchestration_movers.go

```diff
@@ -453,31 +453,55 @@ func (o *datasyncOrchestrator) taskDetailsFromName(ctx context.Context, group, n
 	return nil, nil, apierror.New(apierror.ErrNotFound, "datasync mover not found", nil)
 }
 
-// taskRunsFromTaskArn finds a datasync task runs  based on its group/name and returns information about it
-func (o *datasyncOrchestrator) taskRunsFromTaskArn(ctx context.Context, taskArn string) ([]string, error) {
-	if taskArn == "" {
+// datamoverRunList finds a datasync task runs  based on its group/name and returns information about it
+func (o *datasyncOrchestrator) datamoverRunList(ctx context.Context, group, name string, fullArn bool) ([]string, error) {
```

Comment:
We shouldn't need that `fullArn` bool as this should only return a list of exec id's

Reply:
This method we are using twice 
1. /{account}/movers/{group}/{name}/runs --- with false to get truncated IDs not the full ARN 
2. /{account}/movers/{group}/{name}/runs/{id} --- with True, as ID in this route will come truncated and we need full ARN to get information.